julia/src/compile.jl

```diff
@@ -6,9 +6,8 @@ end
 function get_bridgestan()
```

Comment:
We should update the docstring of this function to clarify the behavior with the artifact

Reply:
Done! This function isn't currently exported.

Reply:
We probably want to export it (might as well in Python/R as well, since it makes the example programs easier to write generically, as you've done here). Maybe under the name `get_bridgestan_path` rather than `get_bridgestan`?

Is there a way to issue some sort of diagnostic (warning, print statement, anything) the when the LazyArtifact is downloaded? 
I might be imagining users who don't exist, but I have a concern that people will download the repository and then not realize the actual source is being used from somewhere else, so it would be nice if we could say something like 
>Downloading the BridgeStan source as an artifact to `$download_location`. If you wish to use an alternative source, use `set_bridgestan_path!()`

Reply:
Yes, I am just now remembering R doesn't even have the compilation utilities yet, so there's certainly nothing to update there. Python can be done separately (we will probably want to find a way to do something similar in that package as well, if we want to ship this on PyPi)